ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/AdminShapeRestController.java

```diff
@@ -36,22 +33,14 @@ private void initBinder(WebDataBinder binder) {
 
 	@GetMapping("/eventstreams/{collectionName}/shape")
 	public ResponseEntity<LdesConfigModel> getShape(@PathVariable String collectionName) {
-		LdesConfigModel shape = service.retrieveShape(collectionName);
+		LdesConfigModel shape = shaclCollection.retrieveShape(collectionName);
 		return ResponseEntity.ok(shape);
 	}
 
 	@PutMapping("/eventstreams/{collectionName}/shape")
-	public ResponseEntity<LdesConfigModel> putShape(@PathVariable String collectionName,
+	public void putShape(@PathVariable String collectionName,
```

Comment:
no return message for PUT ?

Reply:
updated to return input (method only publishes event)